### ISSUE TITLE
Can't set up a Nuke project

### ISSUE THREAD

**Issue by markusamshove** [nuke-build/nuke#110]
Hi,

I'm following your [Getting started](https://nuke.build/getting-started.html) documentation and the first thing I did was installing the `nuke` global tool.

When invoking `nuke` or `nuke :setup` I get the following error:

`Assertion failed: obj != null`

Nothing else is done/shown.

I was then trying to invoke it with `--help` to see if I can make the output a bit more verbose, but this results in the same error.

Since the global tool didn't work for me, I tried using the PowerShell setup, but the [bootstrapping](https://raw.githubusercontent.com/nuke-build/nuke/master/bootstrapping/setup.ps1) file from the docs isn't available any longer.

Is there anything I can do to get a better understanding on whats going wrong?

I'm trying to add a build script for a solution containing net4.5.2 projects (having the new csproj format)

**Comment by markusamshove** [comment]
I cloned Nuke and debugged it against my project and found [this](https://github.com/nuke-build/nuke/blob/develop/source/Nuke.Common/Tooling/NuGetPackageResolver.cs#L117) line to return `null`, which causes [this](https://github.com/nuke-build/nuke/blob/develop/source/Nuke.GlobalTool/Program.Setup.cs#L43) to throw, because of the `NotNull()` requirement.

I'm not sure what I'm missing on my part so that it has to fail, as mentioned I just installed the global tool according to the docs.

The package `Nuke.Common` is not installed in `%USERPROFILE%\.nuget`, which seems to fail this


Edit:

I've manually downloaded the `Nuke.Common` package from `nuget.org` and put the .nupkg in `%USERPROFILE%\.nuget\packages` and also extracted it, but the GlobalTool still seems to not find it

Edit2:

After manually putting the `.nupkg` file in `%USERPROFILE%\.nuget\nuke.common\0.6.0\nuke.common.0.6.0.nupkg` it worked.

Did I miss something when installing to get it correctly set up for me from the start? :)
